stdlib/FileWatching/test/runtests.jl

```diff
@@ -369,60 +369,62 @@ test_monitor_wait_poll()
 test_monitor_wait_poll()
 test_watch_file_timeout(0.2)
 test_watch_file_change(6)
-test_dirmonitor_wait2(0.2)
-test_dirmonitor_wait2(0.2)
-
-mv(file, file * "~")
-mv(file * "~", file)
-let changes = []
-    while true
-        let c
-            Sys.iswindows() && sleep(0.1)
-            @test @elapsed(c = watch_folder(dir, 0.0)) < 0.5
-            push!(changes, c)
-            (c.second::FileWatching.FileEvent).timedout && break
-        end
-    end
-    if F_GETPATH
-        @test 12 < length(changes) < 48
-    else
-        @test 5 < length(changes) < 16
-    end
-    @test pop!(changes) == ("" => FileWatching.FileEvent())
-    if F_GETPATH
-        Sys.iswindows() && @test pop!(changes) == (F_PATH => FileWatching.FileEvent(FileWatching.UV_CHANGE))
-        p = pop!(changes)
-        if !Sys.isapple()
-            @test p == (F_PATH => FileWatching.FileEvent(FileWatching.UV_RENAME))
-        end
-        while changes[end][1] == F_PATH
-            @test pop!(changes)[2] == FileWatching.FileEvent(FileWatching.UV_RENAME)
-        end
-        p = pop!(changes)
-        if !Sys.isapple()
-            @test p == (F_PATH * "~" => FileWatching.FileEvent(FileWatching.UV_RENAME))
-        end
-        while changes[end][1] == F_PATH * "~"
-            @test pop!(changes)[2] == FileWatching.FileEvent(FileWatching.UV_RENAME)
+
+if !((Sys.ARCH == :x86_64) && (Sys.isapple())) #These tests tend to fail a lot on x86-apple
```

Comment:
Whoops, I may have been a little overeager. These fail on aarch64 also, so should be disabled everywhere.

Reply:
I actually haven't seen this fail in FileWatching in aarch64, the only one I've seen was a failure in the PIDFile test.

Reply:
Ok, but the moment we see it, the test is gone ;)